### PR TITLE
 Skip option in Welcome Screen

### LINKED ISSUE #1239: Skip option in the Welcome screen for beta debug builds
**Summary:** 

Skip option in the Welcome screen. There are 5 pages in this screen and as a regular user, whenever i reinstall the app, i find it meaningless to swipe to get to the last screen. 

**Steps to reproduce:** 

skip_welcome

**Expected behavior:** 

Option to **Skip** the tutorial would be great.

### PR DESCRIPTION
**Pull request by tanvidadu**
## Description

Fixes #1239 

Added Skip option in Welcome Screen
## Description

Fixes #1239 

Added Skip option in Welcome Screen

## Screenshot
skipoption

### CODE REVIEW

**Comment by maskaravivek** [comment]
@tanvidadu Please attach a screenshot.

File: app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java [hunk 1]
```diff
@@ -54,10 +54,14 @@ public boolean isViewFromObject(View view, Object object) {
     public Object instantiateItem(ViewGroup container, int position) {
         LayoutInflater inflater = LayoutInflater.from(container.getContext());
         ViewGroup layout = (ViewGroup) inflater.inflate(PAGE_LAYOUTS[position], container, false);
-
-        if (position == PAGE_FINAL) {
+        if( BuildConfig.FLAVOR == "beta"){
             ViewHolder holder = new ViewHolder(layout);
             layout.setTag(holder);
+        } else {
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java]: I dont think it should be part of `else`. Please leave the original `if` check intact.

Comment (tanvidadu) [reply app/src/main/java/fr/free/nrw/commons/WelcomePagerAdapter.java]: In beta flavour we needed to button on each screen of tutorial passed to holder, so I removed if to ensure it is followed in all the layouts inflated. In other versions if condition is still there.

**Comment by psh** [comment]
You know, if the "Skip" button was in a different location on the screen - say, styled as a text link at the top right of the layout - you wouldnt need to put it on every page of the view pager.  Then you would avoid duplication _plus_ it would be a little more subtle.  How about something like this - 

device-2018-03-06-081252

**Comment by tanvidadu** [comment]
Looks good ! I will change it accordingly 👍

**Comment by tanvidadu** [comment]
New mockups 
newtut

**Comment by maskaravivek** [comment]
It works fine on beta builds. The option is visible even on `prodDebug` builds and clicking it does nothing. @tanvidadu you will need to fix that before this can be merged.

**Comment by tanvidadu** [comment]
@maskaravivek @misaochan @psh  I have fixed the above problem.

**Comment by tanvidadu** [comment]
@maskaravivek I have fixed those error. Thanks :)

**Comment by maskaravivek** [comment]
Tested on both beta and prod builds. Works as expected. Thanks for the contribution @tanvidadu. Surely all commons app devs will use this feature everytime they do a fresh install. :D

**Pull request merged**